components/products/ProductsScreenshot.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+const { t } = useI18n();
+
+const data = [
+  {
+    img: '/img/products/liquity.webp',
+    title: t('products.features.liquity.title'),
+    description: t('products.features.liquity.description'),
```

Comment:
@lukicenturi I guess this is what @LefterisJP said during standup that we should remove since it does not make sense?

Reply:
Updated, I made the adjustment, remove the liquity, and remove some duplication so it should looks better